connector-runtime/connector-runtime-core/src/test/java/io/camunda/connector/runtime/core/outbound/ConnectorJobHandlerTest.java

```diff
@@ -343,6 +350,22 @@ void shouldNotFail_MappingNonExistingKeys() {
         assertThat(result).isNotNull();
       }
 
+      @Test
+      void shouldSucceed_MappingFromScalarToContext() {
+        // given
+        var jobHandler = newConnectorJobHandler((context) -> "FOO");
+        var resultExpression = "{processedOutput: response}";
+
+        // when
+        var result =
+            JobBuilder.create()
+                .withResultExpressionHeader(resultExpression)
+                .executeAndCaptureResult(jobHandler);
+
+        // then
+        assertThat(result.getVariables()).isEqualTo(Map.of("processedOutput", "FOO"));
```

Comment:
please confirm this is what is expected when a Connector return "FOO" for instance

Reply:
you can also add a test where the same applies for a scalar value when using a result variable.

Reply:
Isn't it [this](https://github.com/camunda/connectors/blob/9216c5f1e9042098ed6192dc4bbd246c4a9171ed/connector-runtime/connector-runtime-core/src/test/java/io/camunda/connector/runtime/core/outbound/ConnectorJobHandlerTest.java#L137) test?

Reply:
Yes, right. Wasnt aware that its already present.